src/components/pages/StudentsPage/index.tsx

```diff
@@ -65,15 +66,18 @@ export default function StudentsPage({ classroomName }: ClassroomProps) {
   return (
     <main>
       <Typography variant='h4' sx={{ color: 'white', mb: 4 }}>
-        {removedFromClass
-          ? 'Your teacher has removed you from the classroom'
-          : `Hello ${name}! Welcome to your classroom: ${classroomName}`}
+        {`Hello ${name}!`}
       </Typography>
-      {chatInSession && (
-        <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+      <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+        {chatInSession ? (
           <Chatbox socket={socket} chat={chat} setChat={setChat} />
-        </Box>
-      )}
+        ) : (
+          <WelcomeMessage
+            classroom={classroomName}
+            removed={removedFromClass}
```

Comment:
I'd prefer we keep the original variable names. They feel more descriptive to me. 
```suggestion
            classroomName={classroomName}
            removedFromClass={removedFromClass}
```